enderio-conduits-modded/build.gradle.kts

```diff
@@ -46,6 +49,8 @@ dependencies {
     runtimeOnly("mezz.jei:jei-$minecraftVersion-common:$jeiVersion")
     runtimeOnly("mezz.jei:jei-$minecraftVersion-neoforge:$jeiVersion")
 
+    implementation("dev.gigaherz.graph:GraphLib3:$graphlibVersion")
```

Comment:
Was this necessary? The dependency should be transient by including eio conduits.